translations/italian.json

```diff
@@ -39,9 +38,9 @@
 		"DATA COMMANDS": "COMANDI PER I DATI",
 		"DRIVER COMMANDS": "COMANDI PER I DRIVER",
 		"MODERATOR COMMANDS": "COMANDI PER I MODERATORI",
-		"LEADER COMMANDS": "COMANDI PER I LEADER",
+		"ADMIN COMMANDS": "COMANDI PER I LEADER",
```

Comment:
comandi per i amministratori

(i dont speak italian either but google translate gives this for "administrators", and it fits the pattern with "moderators", which is also a real word)